src/main/java/autoLotto/OutputView.java

```diff
@@ -0,0 +1,30 @@
+package autoLotto;
+
+import static autoLotto.Match.*;
+import static java.util.stream.IntStream.*;
+
+public class OutputView {
+
+    public static void outputQuantity(int quantity) {
+        System.out.println(quantity + "개를 구매했습니다.");
+    }
+
+    public static void outputLottos(Lottos lottos) {
+        range(0, lottos.getLottosSize())
+                 .forEach(i -> System.out.println(lottos.getLotto(i)));
+    }
+
+    public static void outputResult(double result, int threeMatch, int fourMatch, int fiveMatch, int sixMatch) {
```

Comment:
결과 값들을 담고 있는 객체를 넘기면 파라미터로 어떨까요?
```suggestion
    public static void outputResult(GameResult gameResult) {
```